config/targets.js

```diff
@@ -1,18 +1,11 @@
 'use strict';
 
-const browsers = [
-  'last 1 Chrome versions',
-  'last 1 Firefox versions',
-  'last 1 Safari versions'
-];
-
-const isCI = !!process.env.CI;
-const isProduction = process.env.EMBER_ENV === 'production';
-
-if (isCI || isProduction) {
-  browsers.push('ie 11');
-}
-
 module.exports = {
-  browsers
+  browsers: [
```

Comment:
At work, I found that allowing `last 1 versions` can lead to unexpected build errors. I set these targets based on:

- http://kangax.github.io/compat-table/es2016plus/
- https://caniuse.com/usage-table

For this app, I think it's okay to drop support for IE 11.

Reply:
I would recommend not moving away from the default Ember blueprints on these targets.

Reply:
Ah, ok. I'll revert them.